apps/wing/src/commands/upgrade.ts

```diff
@@ -24,8 +24,10 @@ export async function upgrade(options: IUpdateOptions) {
     pkg: PJSON,
   });
 
-  log("forcing update info...");
-  await notifier.fetchInfo();
+  if (force) {
```

Comment:
How is this configured?

Reply:
Doesn't look like there is a way. Personally, I think an upgrade subcommand makes the most sense to force it, so I can add that

Reply:
I don't think an upgrade command necessarily makes sense since people might have the Wing toolchain installed in different ways (it might not be via npm -g ...)

Reply:
> Can we also add a way to disable the update check?

Done

> Can we disable update checks when running without a connected TTY?

Done

> Can we disable update checks when running with CI environment variable set?

The library we use already does this

> I don't think an upgrade command necessarily makes sense since people might have the Wing toolchain installed in different ways (it might not be via npm -g ...)

So the written code already kinda handles this and can actually run an install command for you. However, I'm not really a fan of this anyways (I install things with volta, for instance).

I went ahead and deleted most of the updating code and kept it simplified to just to updating and notifying.